helpers/all.js

```diff
@@ -12,22 +10,14 @@ const factory = () => {
     return function(...args) {
         // Take the last arg (which is a Handlebars options object) out of args array
         const opts = args.pop();
-
+        let result;
```

Comment:
Minor nitpick, but I would put this below the comment

Reply:
fixed.

Reply:
fixed.